src/activity_handlers.cpp

```diff
@@ -1129,13 +861,13 @@ void activity_handlers::butcher_finish( player_activity *act, player *p )
         act->set_to_null();
         return;
     }
-
+    // function just for drop yields
+    auto roll_drops = [&]() {
+        factor < -50 ? factor = -50 : factor;
```

Comment:
This appears to be a strange way of writing an `if`.  Probably better just to write the `if` statement.

Reply:
Or something like `factor = std::max( factor, -50 );`